packages/libs/react-components/src/components/Input/Input.tsx

```diff
@@ -16,8 +16,8 @@ export interface IInputProps
   > {
   as?: 'input';
   leadingText?: string;
-  leftPanel?: typeof SystemIcons[keyof typeof SystemIcons];
-  rightPanel?: typeof SystemIcons[keyof typeof SystemIcons];
+  leftPanel?: (typeof SystemIcons)[keyof typeof SystemIcons];
```

Comment:
Do you know what is causing this reformating?

Reply:
It's the running of prettier on my end! Will revert and keep an eye out for these

Reply:
It actually seems to be running with the prettier command upon commiting, so not sure how to undo it!

Reply:
To me this formatting does make sense. It's a bit the same as `(a as any).item`. What is weird is that the parens weren't there before, and now Prettier made them appear. Didn't we run Prettier before for this file?